plugins_src/autouv/wpc_snap_win.erl

```diff
@@ -0,0 +1,759 @@
+%%%-------------------------------------------------------------------
+%%% File    : wpc_snap_win.erl
+%%% Author  :  <dgud@erix.ericsson.se>
+%%% Description : Snap texture (image) to model
+%%%
+%%% Created : 28 May 2003 by Dan Gudmundsson
+%%% Snap Image migration to window mode: 30 Jan 2017 by Micheus
+%%-------------------------------------------------------------------
+%%  Copyright (c) 2002-2011 Dan Gudmundsson, Bjorn Gustavsson
+%%
+%%  See the file "license.terms" for information on usage and redistribution
+%%  of this file, and for a DISCLAIMER OF ALL WARRANTIES.
+%%     $Id$
+
+-module(wpc_snap_win).
+
+-define(NEED_OPENGL, 1).
+
+-include("wings.hrl").
```

Comment:
It can find files in the same directory, so that is not a problem.
When I do something in a plugin I change it to be include_lib,
I don't want new plugin-files to use include.